### ISSUE TITLE
[Feature Request] Separate rule execution failures from accessibility failures in the UI

### ISSUE THREAD

**Issue by karanbirsingh** [microsoft/accessibility-insights-windows#1113]
**Is your feature request related to a problem? Please describe.**

Issue #1041 describes a scenario where an execution error in the rule engine is reported as an accessibility failure. While these are distinguished in Axe.Windows, it seems that Accessibility Insights for Windows reports them similarly in the UI. This is confusing for users.

**Describe the solution you'd like**

Users of Accessibility Insights for Windows should know when a rule fails to execute. This should not be reported as an accessibility error. If possible, information about the error should be reported.

**Comment by RobGallo** [comment]
My suggestion is we surface execution errors (e.g., exceptions) with a new icon (separate from the icon meant to indicate that web content is being scanned)
and store exception information in the a11ytest file. This will enable users to quickly determine whether their issues are accessibility or execution/platform.
It will also enable us to better diagnose issues when they are sent to us.

**Comment by asksep** [comment]
The Accessibility Insights team is awaiting community feedback to identify if this a common issue.
